src/app/admin/components/transcripts/search-completed-transcripts-results/search-completed-transcripts-results.component.html

```diff
@@ -12,7 +12,16 @@
       >
     </td>
     <td>{{ row.requestId }}</td>
-    <td class="whitespace-pre">{{ row.caseNumber }}</td>
+    <td class="whitespace-pre">
+      <a
+        href="#"
+        class="govuk-link"
+        [routerLink]="['/admin/case/', row.caseId]"
+        [queryParams]="{ backUrl: '/admin/transcripts' }"
+      >
+        {{ row.caseNumber }}</a
+      >
+    </td>
```

Comment:
As this is inside the `whitespace-pre` class can you put the variable right next to the HTML tags to prevent any unwanted whitespace. It's something like this.

```suggestion
    <td class="whitespace-pre">
      <a
        href="#"
        class="govuk-link"
        [routerLink]="['/admin/case/', row.caseId]"
        [queryParams]="{ backUrl: '/admin/transcripts' }"
        >{{ row.caseNumber }}</a>
    </td>
```

Reply:
There are a few other places where this needs amending as well.